src/NServiceBus.Core/Features/FeatureStartupTaskController.cs

```diff
@@ -16,6 +16,11 @@ public FeatureStartupTaskController(string name, Func<IServiceProvider, FeatureS
 
         public Task Start(IServiceProvider builder, IMessageSession messageSession)
         {
+            if (Log.IsDebugEnabled)
```

Comment:
Technically because this is not really a hot path we could also remove the if debug guard. Do we have comparison what we do elsewhere in the startup path regarding debug logging?

Reply:
I think the additional check doesn't hurt much in this case, especially since it's a debug log that should never be enabled in production under "normal" circumstances.